include/deal.II/matrix_free/constraint_info.h

```diff
@@ -683,8 +683,9 @@ namespace internal
 
       if (hn_available == true)
         {
-          for (unsigned int v = n_lanes_filled; v < Number::size(); ++v)
-            constraint_mask[v] = unconstrained_compressed_constraint_kind;
+          std::fill(&constraint_mask[n_lanes_filled],
+                    constraint_mask.end(),
```

Comment:
This fails with the Windows compiler because the first argument is a pointer whereas the last one is a proper iterator type.

Reply:
I changed it to `constraint_matrix.begin()+n_lanes_filled` which should hopefully work.